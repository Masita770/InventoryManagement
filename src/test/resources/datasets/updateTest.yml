items:
  - items_id:  1
    category:  "desk"
    item:  "オフィスデスク"
    registration_date:  "2025-03-16 21:01:30"
    deleted_date:  "0000-00-00 00:00:00"
    stocksList:  null
    ordersList:  null