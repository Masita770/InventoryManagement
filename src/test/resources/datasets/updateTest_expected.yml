items:
  - items_id:
    category:  "机"
    item:  "オフィスデスク"
    registration_date:  "2025-03-16 21:01:30"
    deleted_date:  "0000-00-00 00:00:00"
    stocksList:  null
    ordersList:  null